lib/database/schemas.js

```diff
@@ -58,6 +58,10 @@ const content = [
       update: 'CASCADE',
     },
   },
+  {
+    name: 'workflows',
+    type: 'jsonb',
+  },
```

Comment:
Don't like the name `workflows`, can it be something like `approval trail` or `approval` or `audit` or something?